static/docs/get-started/example-pipeline.md

```diff
@@ -167,8 +167,8 @@ intermediate result.
 Second, you should see by now that the actual data is stored in the `.dvc/cache`
 directory, each file having a name in a form of an md5 hash. This cache is
 similar to Git's internal objects store but made specifically to handle large
-data files. DVC is using reflinks, hardlinks and other optimizations to manage
-your actual workspace without copying every time object from/to the cache.
+data files. DVC uses reflinks, hardlinks, or other optimizations to manage your
```

Comment:
the same, we should be emphasizing this. It was really painful decision to disable hardlinks and symlinks. A lot of ppl might be confused and disappointed to see that we are copying their files. We should give a lot of warnings and docs so that they understand right away that there is a way to enable an optimization.

Reply:
Split into special note.